File: duplicate.c
#include<stdio.h>
int main(){
    int a[5] = {1, 2, 3, 4, 5};
    int b[5] = {5, 6, 7, 8, 9};
    int c[10];
    int k = 0;
    int isDuplicate;

    for(int i = 0; i < 5; i++) {
        isDuplicate = 0;
        for(int j = 0; j < 5; j++) {
            if(a[i] == b[j]) {
                isDuplicate = 1;
                break;
            }
        }
        if(!isDuplicate) {
            c[k++] = a[i];
        }
    }


    for(int i = 0; i < 5; i++) {
        isDuplicate = 0;
        for(int j = 0; j < 5; j++) {
            if(b[i] == a[j]) {
                isDuplicate = 1;
                break;
            }
        }
        if(!isDuplicate) {
            c[k++] = b[i];
        }
    }

    printf("Non-duplicate elements: ");
    for(int i = 0; i < k; i++) {
        printf("%d ", c[i]);
    }
    printf("\n");
    return 0;
}

File: duplicate.py
a = [1, 2, 3, 4, 5]
b = [5, 6, 7, 8, 9]
c = []

for i in range(len(a)):
    isDuplicate = False
    for j in range(len(b)):
        if a[i] == b[j]:
            isDuplicate = True
            break
    if not isDuplicate:
        c.append(a[i])
for i in range(len(b)):
    isDuplicate = False
    for j in range(len(a)):
        if b[i] == a[j]:
            isDuplicate = True
            break
    if not isDuplicate:
        c.append(b[i])

print("Non-duplicate elements:", c)
